src/organizations/components/OrgProfileTab.tsx

```diff
@@ -43,105 +49,175 @@ const OrgProfileTab: FC = () => {
+            ></Input>
+            <Button
+              testID="rename-org--button"
+              text="Rename"
+              icon={IconFont.Pencil}
+              onClick={handleShowEditOverlay}
+            />
+          </FlexBox>
+          <FlexBox
+            direction={FlexDirection.Row}
+            margin={ComponentSize.Medium}
+            justifyContent={JustifyContent.SpaceBetween}
+            stretchToFitWidth={true}
+            style={{width: '85%'}}
+          >
+            {[
+              {
+                label: 'Provider',
+                src: convertProviderSymbol(me.quartzMe?.billingProvider),
+              },
+              {label: 'Region', src: me.quartzMe?.regionCode},
+              {label: 'Location', src: me.quartzMe?.regionName},
+            ].map(({label, src}) => {
+              if (!!src) {
+                return (
+                  <FlexBox
+                    key={`org-${label.toLowerCase()}`}
+                    direction={FlexDirection.Column}
+                    margin={ComponentSize.Large}
+                    alignItems={AlignItems.FlexStart}
+                  >
+                    <Heading
+                      element={HeadingElement.H4}
+                      weight={FontWeight.Regular}
+                      style={headingStyle}
+                    >
+                      {label}
+                    </Heading>
+                    <span style={{fontWeight: FontWeight.Light, width: '100%'}}>
+                      {src}
+                    </span>
+                  </FlexBox>
+                )
+              } else {
+                return null
+              }
+            })}
+          </FlexBox>
           <Heading
             element={HeadingElement.H4}
-            style={{marginTop: '16px', marginBottom: '4px'}}
+            style={headingStyle}
             weight={FontWeight.Regular}
           >
-            Organization ID
+            Cluster URL
           </Heading>
-          <div className="code-snippet" data-testid="code-snippet--orgid">
+          <div
+            className="code-snippet"
+            style={stretchStyle}
+            data-testid="code-snippet--userid"
+          >
             <div className="code-snippet--text">
               <pre>
-                <code>{org.id}</code>
+                <code>{me.quartzMe.clusterHost}</code>
               </pre>
             </div>
             <FlexBox
               className="code-snippet--footer"
               margin={ComponentSize.Medium}
+              stretchToFitWidth={true}
             >
               <CopyButton
-                text={org.id}
-                onCopy={generateCopyText('Organization ID', org.id)}
-                testID="copy-btn--orgid"
+                text={me.id}
+                testID="copy-btn--organizationUrl"
+                onCopy={generateCopyText(
+                  'Organization URL',
+                  me.quartzMe.clusterHost
```

Comment:
```suggestion
                  me.quartzMe?.clusterHost
```

Reply:
instead, made that entire part (and small view element with surrounding form field) be present based up existence check of `me.quartzMe?.clusterHost`.